workers/unity/Packages/com.improbable.gdk.core/Tests/Editmode/Subscriptions/StandardSubscriptionTests.cs

```diff
@@ -0,0 +1,84 @@
+using Improbable.Gdk.Core;
+using Improbable.Gdk.Subscriptions;
+using Improbable.Gdk.TestUtils;
+using NUnit.Framework;
+using Unity.Entities;
+using UnityEngine;
+
+namespace Improbable.Gdk.Core.EditmodeTests.Subscriptions
+{
+    /// <summary>
+    ///     This tests the availability constraints for the following subscriptions :
+    ///         - EntityId
+    ///         - ECS Entity
+    ///         - Log Dispatcher
+    ///         - World Commands
+    ///         - ECS World
+    /// </summary>
+    [TestFixture]
+    public class StandardSubscriptionTests : SubscriptionTestBase
```

Comment:
What does standard mean

Reply:
Its what you called the folder `StandardSubscriptionManagers`

Reply:
Oh I forgot I used that word.